### ISSUE TITLE
is there a reason for using jquery Deferred in the async answers?

### ISSUE THREAD

**Issue by jacalata** [rmurphey/js-assessment-answers#104]
I may have missed use of jQuery in other answers, but I feel like it would be cleaner to use straight javascript in the answers for these tests - is this something other people have opinions on?

**Comment by technowar** [comment]
I was using Vanilla JS on Async exercises.

**Comment by jacalata** [comment]
Right, I meant the sample answers use jquery, when it is definitely possible to do them without it.

**Comment by onetwothreebutter** [comment]
I agree...I think the answers should not require jQuery to work.

**Comment by rmurphey** [comment]
I'd welcome a pull request.

**Comment by onetwothreebutter** [comment]
Cool, will do
